cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/UserApi.java

```diff
@@ -138,9 +139,17 @@
     @POST( "user/payment/token" )
     Call<Void> saveTokenData( @Body TokenDataDTO data );
 
+    @PATCH( "user/{id}" )
+    Call<Void> partialUpdate( @Path( "id" ) int id, @Body UserPatchDTO input );
+
+    @PATCH( "user/{id}" )
+    Call<Void> partialUpdate( @Path( "id" ) int id, @Body UserPatchDTO input, @Query( "language" ) String language );
+
+    @Deprecated
     @PATCH( "user/{id}" )
     Call<Void> partialUpdate( @Path( "id" ) int id, @Body Map<String, Object> input );
 
+    @Deprecated
     @PATCH( "user/{id}" )
     Call<Void> partialUpdate( @Path( "id" ) int id, @Body Map<String, Object> input, @Query( "language" ) String language );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)